app/server/methods/users.js

```diff
@@ -41,6 +41,11 @@ Meteor.methods({
         email: emailAddress
       });
 
+      if(enrollmentDocument.groups){
+        enrollmentDocument.groups.forEach(r => {
```

Comment:
Instead of `r`, just use `groupId`. In general, avoid single-letter variable names, as they make it difficult to understand what the variable represents. Using `groupId` here would allow us to use cleaner syntax on the next line:

```js
Permissions.insert({ userId, groupId })
```